sdk/python/requirements.txt

```diff
@@ -1,7 +1,7 @@
-cryptography==3.4.6
+cryptography==36.0.1
```

Comment:
is this _right_ ? o0 is this same package actually?

Reply:
hhmmmm, wth o0

<img width="340" alt="image" src="https://user-images.githubusercontent.com/6421169/151245036-b296aa51-34bc-4d55-bb94-7a56aec8090c.png">

Reply:
(probably a typo going from 3.4 => 3.5 ^^)